src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs

```diff
@@ -20,35 +20,33 @@ namespace Microsoft.PowerShell.Commands
     /// Intended to work against the wide spectrum of "RESTful" web services
     /// currently deployed across the web.
     /// </summary>
-    [Cmdlet(VerbsLifecycle.Invoke, "RestMethod", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096706", DefaultParameterSetName = "StandardMethod")]
+    [Cmdlet(VerbsLifecycle.Invoke, "RestMethod", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2096706")]
     public class InvokeRestMethodCommand : WebRequestPSCmdlet
     {
         #region Parameters
 
         /// <summary>
         /// Gets or sets the parameter Method.
         /// </summary>
-        [Parameter(ParameterSetName = "StandardMethod")]
-        [Parameter(ParameterSetName = "StandardMethodNoProxy")]
+        [Parameter]
         public override WebRequestMethod Method
         {
-            get { return base.Method; }
+            get => base.Method;
```

Comment:
I'd move such changes in separate PR since I have concerns about parameterset changes.

Reply:
What are your concerns, how can I address them?

Reply:
These are functional changes.  It's better to separate them from the style changes. It makes code review easier.

Reply:
I separated them